patterns/atoms/global/colors/_colors.scss

```diff
@@ -0,0 +1,28 @@
+
+.colors__color1 {
```

Comment:
we should be generating color palette based on provided base colors, not just defining them to background-color property. also all colors should be accessible via variables or function calls. check out [this article](http://www.sitepoint.com/using-sass-build-color-palettes/). I would suggest just to leave this component as it is for now.